src/document/operation/changeoperation.js

```diff
@@ -130,6 +134,12 @@ CKEDITOR.define( [
 					value.node.setAttr( newAttr );
 				}
 			}
+
+			return {
+				range: this.range,
+				oldAttr: oldAttr ? oldAttr : undefined,
```

Comment:
Is this really needed? Also I don't think that this should be `undefined` while the operation expected argument is `null` when it is "empty". We should stick with one of them and I propose `null` as for me it passes more data. Then, we should simply return `oldAttr` and `newAttr`. Remember to change docs in `document.Document`.